### PR TITLE
Add a file file for githubjgjucha.md

### PR DESCRIPTION
**Pull request by jgjucha**
This Pull Request adds a bio file for @githubjgjucha as requested in #15 

Hey @tallstuff1964, i would :heart: if you had some :eyes: on this and let me know how you are :+1: with it. :smiley:

### CODE REVIEW

**Comment by jgjucha** [comment]
These changes look good ever all, bu there are a few things that you might want to address.

File: docs/githubjgjucha.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# GitHub Jocelyn Bio
+
+- Name: GitHub Student
+- email: jocelyn.g.jucha@uscis.dhs.gov
+- Occupation: Release Analyst
+- Location: Washington, DC
```
Comment (jgjucha) [inline docs/githubjgjucha.md:6]: There are lots of Ashburn in the :earth_asia:

**Comment by tallstuff1964** [comment]
I don't know, this file looks sketchy.

**Comment by jgjucha** [comment]
It is very interesting

**Comment by jgjucha** [comment]
@klsweet :eyes:

**Comment by jgjucha** [comment]
@lncole :+1:

**Comment by jgjucha** [comment]
@tallstuff1964 @lncole so :confused:

File: docs/githubjgjucha.md (hunk 1 above)
Comment (jgjucha) [inline docs/githubjgjucha.md:6]: Maybe not..

**Pull request merged**